apps/commudle-admin/src/app/feature-modules/listing-pages/public-home-list-experts/components/public-home-list-experts/public-home-list-experts.component.ts

```diff
@@ -8,9 +9,10 @@ import { SeoService } from 'apps/shared-services/seo.service';
 })
 export class PublicHomeListExpertsComponent implements OnInit {
   isMobileView: boolean;
-  constructor(private seoService: SeoService) {}
+  constructor(private seoService: SeoService, private footerService: FooterService) {}
```

Comment:
please check, when we change the component the footer service will change or not